MCLOUD-10863: Release Cloud Tools (#66)
AdobeDocs/commerce-cloud-tools: src/pages/docker/configure/extend-docker-configuration.md

   docker compose -f docker-compose.yml -f docker-compose-dev.yml run deploy bash
   ```

## Composer configuration

You can specify needed Composer version using the env variable `COMPOSER_VERSION`, if you need to clear Composer cache you can use the variable `COMPOSER_CLEAR_CACHE`.
Each version of PHP Docker images has specific Composer version and by default Composer cache is not cleared during starting Docker container from these images.

> The example how you can set needed Composer version and clear Composer cache:

Reviewer: ```suggestion
**To set the Composer version and clear Composer cache**:
```
